CHANGELOG.md

```diff
@@ -1,3 +1,162 @@
+## 5.14.0
+
+### System Improvements
+
+-   (In3luki) Support Babele's `originalName` property in compendium browser search
+-   (Dire Weasel) Fix disabled appearance of NPC resistances header
+-   (DocSchlock) Show Force checkbox on FlatModifier Form when Attribute selected
+-   (DocSchlock) Add Hero Point RollOption on rerolls
+-   (DocSchlock) Add Metagame Setting to ignore the Secret Trait on Checks
+-   (DocSchlock) Add sound emission to Vehicles
+-   (DocSchlock) Add support for Effect Badge Labels that loop to start
+-   (DocSchlock) Add support for up to two divine skills in deity items
+-   (Drental) Add targetOwner option to @Check syntax
```

Comment:
Should use `@Check` to prevent some random user being tagged.

Reply:
Source code doesn't tag github users, but highlighting it in a comment might. :)